<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>log_parser</string>
  </property>
  <widget class="QWidget" name="LogParserWidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>856</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1356</width>
        <height>80</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>274</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>274</width>
         <height>80</height>
        </size>
       </property>
       <property name="title">
        <string>Log_Type</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkBox_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="checkBox_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>274</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>274</width>
         <height>80</height>
        </size>
       </property>
       <property name="title">
        <string>Log_Type</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox_18">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkBox_13">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="checkBox_14">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_15">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBox_16">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="checkBox_17">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>80</height>
        </size>
       </property>
       <property name="title">
        <string>Log_Filter</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="editLogFilter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>700</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLogFilter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>process</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTableWidget" name="TableWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="open_log"/>
    <addaction name="adb_connect"/>
    <addaction name="recently_files"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="Tool">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="af_debugger"/>
    <addaction name="persist_settings"/>
   </widget>
   <widget class="QMenu" name="About">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="version"/>
   </widget>
   <addaction name="File"/>
   <addaction name="Tool"/>
   <addaction name="About"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="open_log">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open Log</string>
   </property>
  </action>
  <action name="adb_connect">
   <property name="text">
    <string>Adb Connect</string>
   </property>
  </action>
  <action name="recently_files">
   <property name="text">
    <string>Recently Files</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="af_debugger">
   <property name="text">
    <string>AF_debugger</string>
   </property>
  </action>
  <action name="persist_settings">
   <property name="text">
    <string>Persist settings</string>
   </property>
  </action>
  <action name="version">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>open_log</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>myExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>adb_connect</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>adbConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recently_files</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>recentlyFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>af_debugger</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>afDebugger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>persist_settings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>persistSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>version</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>version()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLogFilter</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>logFilterOnClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1219</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editLogFilter</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>logFilterOnClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>874</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openLog()</slot>
  <slot>adbConnect()</slot>
  <slot>recentlyFiles()</slot>
  <slot>afDebugger()</slot>
  <slot>persistSettings()</slot>
  <slot>version()</slot>
  <slot>myExit()</slot>
  <slot>logFilterOnClick()</slot>
 </slots>
</ui>
